### ISSUE TITLE
Official Classic Release

### ISSUE THREAD

**Issue by vallantv** [the-rebel-Mermaid/BlizzMove#12]
I believe this addon works for Classic but would you consider doing an official release for it?

**Comment by zanony** [comment]
Is there a link on curse how to do this?

**Comment by vallantv** [comment]
I can test it on classic and see what would need to be changed. Mostly though it's setting the TOC to be a classic version and doing a classic release via curseforge.

**Comment by Numynum** [comment]
when you manually upload files to curseforge you can set the supported classic version
I'm assuming that the automatic release through git tags will select this, if the classic version is there in the TOC
I think @zaCade has done some work to support classic, so maybe he'd have some ideas about this?

**Comment by zaCade** [comment]
I've just made the addon code support classic, as for automatic uploading ect.
As for uploading, i have no idea on how that works. But the addons that do support classic, have their own project pages on curse, but it might also be that it auto selects.

**Comment by Numynum** [comment]
https://twitter.com/WeakAuras/status/1166495588856340480
seems you need a separate project for classic indeed
a bit silly, but ow well

**Comment by vallantv** [comment]
https://github.com/vallantv/MBBClassic As long as the toc reflects classic the build hooks for for curseforge will make a "classic" release.

**Comment by vallantv** [comment]
I'd be more than happy to do this. Can you all make a classic branch or another repo that I can do a pull request against?

**Comment by zanony** [comment]
I had made made BlizzMove support classic back in the classic beta... 

As is evident from the: 
```
local tocversion = select(4, GetBuildInfo())
if tocversion >= 20000 then ...
```
all it needs needs is another automated build for the other classic toc version.

Another brach is an option but kind of a bad one as it really is only a release issue.
I have asked about it here in wowinterface as the packer I use for the wowinterface supports it: https://www.wowinterface.com/forums/showthread.php?t=55801&page=4

**Comment by zanony** [comment]
> I've just made the addon code support classic, as for automatic uploading ect.
> As for uploading, i have no idea on how that works. But the addons that do support classic, have their own project pages on curse, but it might also be that it auto selects.

An own project page might be a good approach but I will still need to figure out how to have 2 project id's (## X-WoWI-ID: 23875) in the toc then.

**Comment by zaCade** [comment]
Wouldn't using the @retail@ things be useful for that? As in besides adding the interface versions separately add the project stuff like that too?

**Comment by vallantv** [comment]
I think it'd be easiest to just create a second curseforge project. If you do manual releasing i think you can do it on the same project. If you want to use github hooks it will identify the retail vs classic by looking at the toc but i don't know if you can have two separate build hooks for curseforge as you'd need a separate branch or separate repo for the classic version. The classic version should literally be a one and done unless folks want to add more features. I don't think the API's will change for classic unless they do a classic plus or launch BC at some point.

**Comment by Numynum** [comment]
most features requested (and added) for retail would work on classic as well though
only adding new frames is irrelevant for classic

**Comment by the-rebel-Mermaid** [comment]
> Wouldn't using the @Retail@ things be useful for that? As in besides adding the interface versions separately add the project stuff like that too?

Yes I have added them for that. They are for the bigwigs packer.
BlizzMove is currently using the curse packer for curse and travis-ci for WoWI.

I looked into how to use the [bigwigs packer](https://github.com/BigWigsMods/packager) with travis-ci but seems like most have abandoned that in favor of GitHub workflows. So I will look into that: https://github.com/WeakAuras/WeakAuras2/blob/master/.github/workflows/build.yml

Seams like most have switched
